src/OrchardCore.Modules/OrchardCore.AuditTrail/Services/DateTimeParser.cs

```diff
@@ -67,22 +67,6 @@ public override string ToString()
             => $"{(String.IsNullOrEmpty(Operator) ? String.Empty : Operator)}{DateTime.ToString("o")}";
     }
 
-    public class DateNode2 : OperatorNode
```

Comment:
I would have to understand why I added this, it was for a reason. So I disagree with removing it.

Reply:
> So I disagree with removing it.

Take your time to check this, I notice that the `DateNode` that you added is based on `DateTime` which makes issues with offset, now I need to asked you what are the reasons for adding this ;)

Reply:
This method implies that the date passed in a UtcDateTime. Just add comments on the method.

Reply:
@Skrypt can you elaborate?

Reply:
What I'm saying is that if DateNode2 uses a DateTime as a parameter and that this DateTime should always be as UTC then you can also validate it inside the DateTime2 constructor instead to keep what we had. If we remove DateNode2 then we need to keep it and mark it as Obsolete instead. But, I assume here that this is used only by the AuditTrail so the changes you made are fine. So, up to @deanmarcussen to illustrate his point.

Reply:
I'm still waiting for @deanmarcussen to clarify why `DateTime2` is here, it's fine to mark it as `obsolete` for now